### ISSUE TITLE
ADD ANTITEAMING

### ISSUE THREAD

**Issue by nobsterr** [Barbosik/MultiOgar#122]
Add antiteaming pls

**Comment by MastaCoder** [comment]
Anti-teaming is pointless.

**Comment by NatsuTheGreat** [comment]
It's better to add back eating inefficiency.

**Comment by darknight99x** [comment]
@NatsuTheGreat I agree, it's becoming a real pain seeing teamers all around my server.

**Comment by Barbosik** [comment]
the problem with eating inefficiency is that it requires much more calculations and leads to worse server performance.
